src/sass/form-render.scss

```diff
@@ -46,4 +46,28 @@
       height: auto;
     }
   }
+.signature-container {
+  border: 2px solid #000; 
+  padding: 10px; 
+  margin-bottom: 10px; 
+  border-radius: 5px; 
+}
+
+.signature-pad {
+  width: 100%; 
+  height: 100%; 
+}
+.signature-pad.custom-style {
+  border: 2px solid #000;
+  background-color: #f9f9f9;
+}
+.clear-button {
+  top: 10px; /* Adjust as needed */
```

Comment:
this looks repeated from above. maybe signature style can be its own file that is imported where needed.